cypress/integration/channel/edit_message_spec.js

```diff
@@ -109,4 +109,46 @@ describe('Edit Message', () => {
             });
         });
     });
+    it('M13542 Open edit modal immediately after making a post', () => {
+        // 1. Login as "user-1" and go to /
+        cy.login('user-1');
+        cy.visit('/');
+
+        // 2. Post message "Hello"
+        cy.get('#post_textbox').type('Hello').type('{enter}');
+
+        // 3. Post a second message "World!"
+        // 4. Hit 'enter' key and then immediately (as FAST as you can) hit 'arrow-up' to open the edit box
+        cy.get('#post_textbox').type('World!{enter}{uparrow}');
+        cy.get('#edit_textbox').
+            invoke('val').
+            then((val) => { // eslint-disable-line
+                const editText = val;
+                if (editText === 'Hello') {
+                    // 5. If pressed while the new post was still pending, observe the edit box opens for the first post
+                    cy.get('#editPostModal').should('be.visible');
+                    cy.get('#edit_textbox').type(' New message{enter}');
+                    cy.get('#editPostModal').should('be.not.visible');
+
+                    // Check the first post and verify that it contains new edited message.
+                    cy.get('.post-message__text').eq(-2).should('contain', 'Hello New message');
```

Comment:
This is not working for me.  Should this be `eq(1)` and the other one below is `eq(0)`?  Same case in the else if condition.

Reply:
<img width="260" alt="Screenshot 2019-04-10 at 11 46 21" src="https://user-images.githubusercontent.com/1737933/55873108-ad22e000-5b86-11e9-915f-313456c8426e.png">

On editing any text, we get a new text stating '(edited)' as shown in the attached image.
That constitutes a <div> in the block and thus I've altered the indexes to take that into account.

I've corrected the numbers and verified the successful passing of the test now. 
(I had gotten the index wrong before)
@saturninoabril Please have a look

Reply:
The negative index does not give me a consistent result.  On the attached screenshot, the post ID is different at -3 index between actual and expected.
![Screen Shot 2019-04-10 at 9 09 34 PM](https://user-images.githubusercontent.com/5334504/55881391-247f6080-5bd5-11e9-9cc3-9e725462b403.png)

Using index of 1 and 0 are consistent, at least on me.
![Screen Shot 2019-04-10 at 9 14 08 PM](https://user-images.githubusercontent.com/5334504/55881668-aa031080-5bd5-11e9-8e92-e21cc40624cb.png)

Reply:
If it seems to be inconsistent, I'll change it as per your suggestion